src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -310,6 +278,23 @@ const formConfig = {
               'ui:options': {
                 updateSchema: (formData, formSchema) => {
                   const fs = formSchema;
+                  if (
+                    get(
+                      'certifierRelationship.relationshipToVeteran.applicant',
+                      formData,
+                    )
+                  ) {
+                    // If the certifier is also an applicant, pre-populate first app slot with certifier info:
+                    populateFirstApplicant(
+                      formData,
+                      SIGNER_REL_PATH, // Used to verify we only ever apply this fn in one location
+                      formData.certifierName,
+                      formData.certifierEmail,
+                      formData.certifierPhone,
+                      formData.certifierAddress,
+                    );
+                  }
```

Comment:
This is where we handle pre-filling the certifier's information into the first applicant in list loop IF they choose that they are an applicant on this `certifierRelationship` question.